tests/semantic/agg_checks3/agg_checks3.dl

```diff
@@ -0,0 +1,24 @@
+// Souffle - A Datalog Compiler
+// Copyright (c) 2020, The Souffle Developers. All rights reserved
+// Licensed under the Universal Permissive License v 1.0 as shown at:
+// - https://opensource.org/licenses/UPL
+// - <souffle root>/licenses/SOUFFLE-UPL.txt
+
```

Comment:
What is this test for? Include a comment that describes the purpose of the test.